### DISCUSSION TITLE
Syntax highlighting in code blocks

### DISCUSSION THREAD

**Discussion by dukemagus** [forem/forem#16941] [Feature Requests]
Right now many posts have interesting code snippets, but no highlighting, which makes the reading a bit more tiresome (specially if you're used to modern IDEs, that color code most elements to make separation easier.)

right now the syntax highlighter repo with most stars on github is highlight.js . since it has no dependencies, it shouldn't make forem less maintainable

**Comment by wescpy** [comment]
I want to highlight lines in triple-backticked codeblocks. Can Forem adopt the same syntax as ReType to make it easier for content creators? Check out this easy syntax: <https://retype.com/components/code-block/#line-highlighting> Below are samples for single lines as well as multi-line code highlighting. Run the "demo" for each sample to see how the highlighting is rendered.
